### PR TITLE
Add rubocop-govuk with some other TLC

### PR DESCRIPTION
**Pull request by kevindew**
This repo hasn't had linting before (!) so I've installed that and fixed the violations.

I've also resolved a number of consistency issues and reduced the number of files in this repo.

### CODE REVIEW

File: config/initializers/gds_sso.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+GDS::SSO.config do |config|
+  config.user_model   = "User"
+  config.oauth_id     = ENV["OAUTH_ID"] || "cffdb8d8df2b3c8ec8e156ff4763d6315603bb89a812ce297691d8536510da38"
+  config.oauth_secret = ENV["OAUTH_SECRET"] || "ec4d814b1d2942da98e0a053c0e2fe5d6d85f1700a4465701b69774b3cdc6655"
+  config.oauth_root_url = Plek.new.external_url_for("signon")
```
Comment (1pretz1) [inline config/initializers/gds_sso.rb:5]: Nitpick: the last '=' isn't lined up with the rest of them.

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -31,52 +22,16 @@
...
     # a real object. This is generally recommended, and will default to
     # `true` in RSpec 4.
     mocks.verify_partial_doubles = true
   end
 
-  # These two settings work together to allow you to limit a spec run
-  # to individual examples or groups you care about by tagging them with
-  # `:focus` metadata. When nothing is tagged with `:focus`, all examples
-  # get run.
-  config.filter_run :focus
```
Comment (1pretz1) [inline spec/spec_helper.rb:47]: Good spot that we were not using this feature 👍

**Review by 1pretz1 — APPROVED** [review]
Great bit of housekeeping! 💯

File: config/initializers/gds_sso.rb (hunk 1 above)
Comment (kevindew) [reply config/initializers/gds_sso.rb:5]: Thanks - sorted

**Pull request merged**